util/tr_openstudio.rb

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/ruby
```

Comment:
Why is this named tr_openstudio?  Maybe something like test_measures would be more descriptive?